types/jest/index.d.ts

```diff
@@ -467,6 +497,8 @@ declare namespace jest {
          * Matches any received string that contains the exact expected string
          */
         stringContaining(str: string): any;
+
+        not: InverseAsymmetricMatchers;
```

Comment:
I'm trying to think of some helpful documentation here to this object. I can find an official description for it.